.github/ISSUE_TEMPLATE/documentation_request.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Documentation request
+about: Missing documentation or suggest an improvement
+
+---
+
+<!--
+Thanks for submitting this documentation request. Please fill the template
+below, otherwise we will not be able to process this request.
+
+Open an issue with this template if you think documentation regarding a certain
+aspect(s) is lacking or missing, or if you have an idea on how
+to improve the documentation.
+-->
+
+#### Summary
+<!-- Summarize the suggestion in a few sentences: -->
```

Comment:
If you call this a "documentation request", then also use "request" here, instead of "suggestion"